### PR TITLE
LPS-154647 Add test WalkthroughPopover#IntermediateStepHasPreviousAndNextStepPresent

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Create automated tests for Walkthrough component. 

**Test Plan**
Given: frontend-js-components-sample-web deployed to test page
And Given: Navigate to Walkable tab on sample portlet
And Given: popover mode
When: on 1st step popover
And When: Click on next steps until ~final step~ second to last step (final step contains Close instead of OK button)
Then: Able to reach ~final step 5~ second to final step
And Then: primary button is present
And Then: secondary button previous step is present

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-154647

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Just started reviewing :)

Test Plan adjustment looks good to me

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/Walkthrough.macro [hunk 1]
```diff
@@ -14,4 +14,10 @@ definition {
 			locator1 = "Walkthrough#POPOVER_TITLE");
 	}
 
+	macro goToSpecificStep {
+		while (IsElementNotPresent(key_title = "Step ${key_step}", locator1 = "Walkthrough#POPOVER_TITLE")) {
+			Click(locator1 = "Button#OK");
+		}
+	}
```
Comment (john-co) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/Walkthrough.macro:21]: cool

**Comment by susanchen3** [comment]
going to fix rebase error first...

**Comment by susanchen3** [comment]
ci:forward

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**